Added fixed aperture flux target measure

thuiop opened the pull request:
Closes #342 

New target measure in the metrics module, for measuring the flux using a fixed circular aperture with sep.

aguinot commented on btk/metrics.py:
```diff
@@ -112,6 +114,37 @@ def meas_ksb_ellipticity(image, additional_params):
+    verbose = additional_params["verbose"]
+
+    bkg = sep.Background(image[meas_band_num])
+    catalog = sep.extract(image[meas_band_num], 1.5, err=bkg.globalrms)
```
If you derive the background RMS from postage stamp I don't think the value will be reliable enough. I think it would be better to take the true value used in the simulation.

aguinot replied:
If you are using small postage stamps I don't think you will be able to get a good estimate. I would rather take the true noise from the simulation.

aguinot commented on btk/metrics.py:
```diff
@@ -112,6 +114,37 @@ def meas_ksb_ellipticity(image, additional_params):
+    flux, fluxerr, flag = sep.sum_circle(
+        image[meas_band_num],
+        catalog["x"],
+        catalog["y"],
```
I am not sure about this for the "reference" value. I think I would also take the true value from simulation for the object positions.

aguinot commented on btk/metrics.py:
```diff
@@ -112,6 +114,37 @@ def meas_ksb_ellipticity(image, additional_params):
+        catalog["x"],
+        catalog["y"],
+        filt.psf_fwhm.to_value("arcsec"),
+        err=bkg.globalrms,
```
Same comment as for the detection.

aguinot commented on btk/metrics.py:
```diff
@@ -112,6 +114,37 @@ def meas_ksb_ellipticity(image, additional_params):
+        image[meas_band_num],
+        catalog["x"],
+        catalog["y"],
+        filt.psf_fwhm.to_value("arcsec"),
```
Maybe not require right now but it could be nice to be able to parametrize this.

aguinot replied:
Should be in pixel

ismael-mendoza commented:
1. fix pixel scale
2. compare apeture flux photometry (sextractor vs scarlet), scatter plot
3. documentation update
4. change labels of figures

thuiop commented:
I fixed the pixel scale issue and updated the doc ; I am merging this now and will open another PR for the plot (there are a few things I need to change, and I'll create a new notebook for reproducibility).
